tests/units/anta_tests/test_snmp.py

```diff
@@ -152,4 +152,41 @@
             "messages": ["SNMP contact is not configured."],
         },
     },
+    {
+        "name": "success",
+        "test": VerifySnmpLogging,
+        "eos_data": [
+            {
+                "logging": {
+                    "loggingEnabled": True,
+                    "hosts": {
+                        "192.168.1.100": {"port": 162, "vrf": ""},
+                        "192.168.1.101": {"port": 162, "vrf": "MGMT"},
+                        "snmp-server-01": {"port": 162, "vrf": "MGMT"},
```

Comment:
can you share the json output from device for hostname

Reply:
```
leaf1-dc1#show snmp | json
{
    "chassis": {
        "chassisId": "C2BB56D511CAB40E40B2DCD1E9D3A9C9"
    },
    "contact": {
        "contact": "\"Network Admin\""
    },
    "location": {
        "location": "\"Data Center A\""
    },
    "counters": {
        "serviceEnabled": true,
        "inPkts": 0,
        "inVersionErrs": 0,
        "inBadCommunityNames": 0,
        "inBadCommunityUses": 0,
        "inParseErrs": 0,
        "inRequestVars": 0,
        "inSetVars": 0,
        "inGetPdus": 0,
        "inGetNextPdus": 0,
        "inSetPdus": 0,
        "outPkts": 5,
        "outTooBigErrs": 0,
        "outNoSuchNameErrs": 0,
        "outBadValueErrs": 0,
        "outGeneralErrs": 0,
        "outGetResponsePdus": 0,
        "outTrapPdus": 5,
        "outTrapDrops": 0,
        "groupLength": 3,
        "numberOfGroups": 9,
        "numberOfViews": 0
    },
    "logging": {
        "loggingEnabled": true,
        "hosts": {
            "192.168.1.100": {
                "port": 162,
                "vrf": ""
            },
            "leaf1-dc1": {
                "port": 162,
                "vrf": ""
            },
            "spine": {
                "port": 162,
                "vrf": ""
            },
            "spine1-dc1.fun.aristanetworks.com": {
                "port": 162,
                "vrf": ""
            }
        }
    },
    "vrfs": {
        "snmpVrfs": [
            "default"
        ]
    },
    "srcIntf": {
        "sourceInterfaces": {
            "default": "Management0",
            "MGMT": "Ethernet1"
        }
    },
    "misc": {
        "transmitMsgMaxSize": 65536
    },
    "enabled": true,
}
```